pkg/cmd/init/cmd.go

```diff
@@ -54,6 +54,13 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 		"If set, the command will initialize the OCM control plan in foreground.")
 	cmd.Flags().StringVarP(&o.output, "output", "o", "text", "output foramt, should be json or text")
 	cmd.Flags().BoolVar(&o.singleton, "singleton", false, "If true, deploy singleton controlplane instead of cluster-manager. This is an alpha stage flag.")
+	cmd.Flags().StringVar(&o.operatorResourceQosClass, "operator-resource-qos-class", "Default", "the resource QoS class of the cluster manager operator. Can be one of Default, BestEffort or ResourceRequirement.")
+	cmd.Flags().StringToStringVar(&o.operatorResourceLimits, "operator-resource-limits", nil, "the resource limits of the cluster manager operator, for example: cpu=800m,memory=800Mi")
```

Comment:
could we also update the flag for operator. Actually, why do we need to set operator separately? Can we just use the same resource-limits flag also for operator?

I am thinking we can start with using the same flat for both operator and all other containers. We could add flat specifically for operator later if there is a real requirement, WDYT?

Reply:
I am OK with that.
Updated.